src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/advanced/AddressComponent.tsx

```diff
@@ -248,7 +242,7 @@ export function AddressComponent(props: IAddressComponentProps) {
             onBlur={onBlurField.bind(null, AddressKeys.careOf, careOf)}
             readOnly={props.readOnly}
           />
-          {allValidations ?
+          {allValidations && allValidations[AddressKeys.careOf] ?
```

Comment:
could be simplified with` allValidations?[AddressKeys.careOf]`

Reply:
I'm not getting that to work.

Reply:
Sorry, missing `.`

Correct syntax is `allValidations?.[AddressKeys.careOf]`